auth-web/src/components/auth/common/PADInfoForm.vue

```diff
@@ -243,6 +246,12 @@ export default defineComponent({
       })
     }) as unknown) as PADInfoFormState
 
+    const getCanDirective = () => {
```

Comment:
Shouldn't we just modify the v-can directive so we can toggle it with a condition?